### ISSUE TITLE
Intel page does not clearly state what OSs are supported

### ISSUE THREAD

**Issue by Craig-Macomber** [GameTechDev/PresentMon#143]
I'm not sure this is the proper place to report this, but as Intel Devs contribute here, hopefully its enough.

This repo is very clear about it being Windows specific which is great, but https://game.intel.com/story/intel-presentmon/ does not mention anywhere that its Windows specific, and the only mention of Windows is in "Broad API Support" where it notes "Both Windows 11 and Windows 10 are supported." This is well below the first download button, and isn't really saying its Windows only.

As a Linux gamer, this is not a great experience. I think the page should list somewhere near the top and/or download button that this is Windows specific. Initially I thought that button might take me to a download page to select a version but it doesn't.

While on the subject of that page, it would be nice if the "Multi-vendor Support and Open Source" section linked to the source. There is a link to the sdk in a different section, but its not clear if the SDK is the same repo as the application (its common for those to be different). That Multi-vendor section also might want to mention that its Windows only, since there are OS vendors, and it could be misinterpreted.

Anyway, thanks for a very neat tool: while I can't use it on my gaming setup, I have benefited from reviewers using it.

**Comment by LethalManBoob** [comment]
Would love this for linux

**Comment by tkg-codes** [comment]
Yeah, I wanted to check out more about my GPU vs CPU stuff while using native Linux games vs Proton, but I saw no platform support. I assumed - hopefully - that the download button will use my OS. Nope. Windows-only.

Too bad, it looks like an awesome tool.

I hope for Linux support, but at least the supported platforms listed would be great.

**Comment by NineMeowICT** [comment]
Linux support would be great.
I have used xpu-smi+mangohud for a while, but xpu-smi cannot support consumer level ARC well.

**Comment by YamiYukiSenpai** [comment]
Just wanna bump that I'd love to see this support Linux as well
